great-idea/index.html

```diff
@@ -14,10 +15,112 @@
+<body>
+  <div class="index-container">
+    <header>
+      <div class="nav-content-container">
+        <nav class="nav-bar">
+          <section class="nav-links">
+            <a href="services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </section>
+          <section class="nav-img">
+            <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+          </section>
+
+
+        </nav>
+      </div>
+
+      <div class="header-content-container">
+        <section class="header-content">
+          <div class="header-title">
+            <h1>
+              Innovation <br>
+              On <br>
+              Demand
+            </h1>
+
+            <button class="get-started">Get Started</button>
+
+          </div>
+          <div class="header-img">
+            <img src="img/header-img.png" alt="Image of a code snippet.">
+          </div>
+
+
+
+
+        </section>
+      </div>
+
+    </header>
+    <div class="top-content-container">
+      <section class="top-content">
+        <h2>Features</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+      <section class="top-content-2">
+        <h2>About</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+      </section>
+    </div>
+
+
+
+    <div class="img-container">
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    </div>
+
+    <div class="bottom-content-container">
+      <section class="bottom-content">
+        <h2>Services</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+      <section class="bottom-content-2">
+          <h2>Product</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse
+            eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+      <section class="bottom-content-3">
+          <h2>Vision</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse
+            eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+     
+    </div>
```

Comment:
same can be applied about reversing the div and section nesting relationship. div should be the child of the section. One section and 3 divs. the structure and indentation is used well.